### PR TITLE
Social Links at bottom

### LINKED ISSUE #2261: MainActivity Unexpected Layout
**Actual Behaviour**

When FossAsia description is minimised layout looks awkward from bottom

**Expected Behaviour**

Layout should be even and user friendly

**Steps to reproduce it**

minimize the fossasia description

**Screenshots of the issue**

ezgif com-video-to-gif 4

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by jindalAnuj**
Fixes #2261. 
MainActivity Unexpected layout  #2261]

Changes: [moved social links recyclerview to relative layout in fragment about.]

ezgif com-video-to-gif 6

### CODE REVIEW

**Comment by jindalAnuj** [comment]
@iamareebjamal Please review my PR

**Comment by mayank8318** [comment]
@jindalAnuj How are snackbars displayed now? Did you test that?

**Comment by jindalAnuj** [comment]
@mayank8318 snakbar are just as it was yes i test that

File: android/app/src/main/res/layout/fragment_about.xml [hunk 1]
```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/fragment_about.xml:2]: What's the use of RelativeLayout here

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/layout/fragment_about.xml (hunk 1 above)
Comment (jindalAnuj) [reply android/app/src/main/res/layout/fragment_about.xml:2]: relative layout is used to keep social link recyclerview at parent bottom

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**